Microsoft.Toolkit.Uwp.UI.Controls/Microsoft.Toolkit.Uwp.UI.Controls.csproj

```diff
@@ -2,6 +2,11 @@
   <PropertyGroup>
     <TargetFrameworks>uap10.0.17763</TargetFrameworks>
     <Title>Windows Community Toolkit Controls</Title>
+    <Description>
+      This library provides Controls, Panels, ItemsRepeater Layouts, and various other classes &amp; helpers for XAML UI development with UWP. It is part of the Windows Community Toolkit.
+
+      This package provides no actual code and is only an aggregate of its dependencies. You can find out how to optimize your app after development at https://aka.ms/wct/optimize
+    </Description>
     <IncludeBuildOutput>false</IncludeBuildOutput>
   </PropertyGroup>
 
```

Comment:
@RosarioPulella we need to include the readme.txt file here as well in the `ItemGroup` below, like we do in the props file, but just here in this csproj directly: https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Directory.Build.props#L106

```xml
<ItemGroup>
  <None Include="$(MSBuildThisFileDirectory)readme.txt" Pack="true" PackagePath="\"/>
```